core/src/main/java/io/confluent/rest/Application.java

```diff
@@ -152,8 +153,14 @@ public Server createServer() throws RestConfigException {
     ResourceConfig resourceConfig = new ResourceConfig();
 
     Map<String, String> metricTags = getMetricsTags();
+    Map<String, String> configuredTags = getConfiguration().getMap(RestConfig.METRICS_TAGS_CONFIG);
 
-    configureBaseApplication(resourceConfig, metricTags);
+    Map<String, String> combinedMetricsTags = new HashMap<>(metricTags);
```

Comment:
looks like metricTags is only used here, so maybe call getMetricsTags() directly here?